src/main/java/org/opensearch/ad/constant/CommonErrorMessages.java

```diff
@@ -94,6 +94,8 @@ public static String getTooManyCategoricalFieldErr(int limit) {
     public static String FAIL_TO_GET_STATS = "Fail to get stats";
     public static String FAIL_TO_SEARCH = "Fail to search";
 
+    public static String FAIL_TO_GET_NODE_STATS = "Fail to get node stats";
```

Comment:
Nit 1. I know you're coping from the previous error messages on lines 85-93, but this should be "failed" rather than "fail".  (As should all the rest.) 

Nit 2. All the others are together without whitespace in between them, this one isn't special.  Save that CR byte from the file size!